### ISSUE TITLE
Empty error

### ISSUE THREAD

**Issue by Msirkovsky** [njpatel/grpcc#42]
I have a problem with empty error: 
I call:
```
 client.getAllMetadata({}, pr)
```
and I recevied empty error:

```
 Error:  { Error
    at C:\user\AppData\Roaming\npm\node_modules\grpcc\node_modules\grpc\src\node\src\client.js:554:15 code: 12, metadata: Metadata { _internal_repr: {} } }
```

Is the error from gRPC endpoint or grpcc tool? Thank you.

**Comment by Msirkovsky** [comment]
the error was at grpc endpoint. It returned status `Unimplemented` with no details. 
It will be great if this status will be shown in grpcc tool too.

**Comment by njpatel** [comment]
Yeah, i want to support grpc errors soon, so we can detect and print them in a nicer fashion. I'll keep this bug as a tracker for that. Thanks.
